softioc/extension.c

```diff
@@ -205,3 +198,59 @@ void EpicsPvPutHook(struct asTrapWriteMessage *pmessage, int after)
         /* Just save the old value for logging after. */
         pmessage->userPvt = value;
 }
+
+static PyObject *install_pv_logging(PyObject *self, PyObject *args)
```

Comment:
Oh, this is where pv logging has gone.

Don't forget, still need to be able to initialise and enable PV logging by simply doing `import softioc.pvlog`.